long_cat.py

```diff
@@ -0,0 +1,34 @@
+import string
+from collections.abc import Generator
+
+
+def words(text: str) -> Generator[str]:
+    """
+    This generator receives text, splits it in to words without punctuations.
+    :param text: The text.
```

Comment:
Variable names should be expressive and not generic. 
https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0